frontend/src/components/eMIB/EmailPreview.jsx

```diff
@@ -1,30 +1,114 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
 
 const styles = {
+  //buttons
   button: {
-    width: 180,
-    marginBottom: 10,
-    marginLeft: 20,
-    marginRight: 20,
-    textAlign: "center",
+    width: 155,
+    textAlign: "left",
     borderRadius: 4,
     padding: 6,
-    border: "2px solid #009fae"
+    border: "2px solid #00565E",
+    cursor: "pointer"
+  },
+  buttonSelectedBackground: {
+    backgroundColor: "#00565E"
+  },
+  buttonReadBackground: {
+    backgroundColor: "#F5F5F5"
+  },
+  buttonUnreadBackground: {
+    backgroundColor: "white"
+  },
+  buttonSelectedText: {
+    color: "#D3FCFF"
+  },
+  buttonUnselectedText: {
+    color: "black"
+  },
+  //li
+  li: {
+    listStyleType: "none"
+  },
+  //subject line
+  subjectSelected: {
+    color: "white"
+  },
+  subjectUnselected: {
+    color: "#00565E"
+  },
+  subjectRead: {
+    fontWeight: "normal"
+  },
+  subjectUnread: {
+    fontWeight: "bold"
   }
 };
 
 class EmailPreview extends Component {
   static propTypes = {
     email: PropTypes.object.isRequired,
-    clickFunction: PropTypes.func.isRequired
+    selectEmail: PropTypes.func.isRequired,
+    isRead: PropTypes.bool.isRequired,
+    isRepliedTo: PropTypes.bool.isRequired,
+    isSelected: PropTypes.bool.isRequired
   };
+
   render() {
+    //READ/UNREAD CHECK
+    //defaults, or if unread
+    let buttonBackgroundColor = styles.buttonUnreadBackground;
+    let imgSrc = <i className="fas fa-envelope" />; //emailUnread;
+    let divId = "unread-email-preview";
+    let subjectIsRead = styles.subjectUnread;
+    if (this.props.isRead) {
+      //if it is read
+      buttonBackgroundColor = styles.buttonReadBackground;
+      subjectIsRead = styles.subjectRead;
+      imgSrc = <i className="far fa-envelope-open" />; //emailRead;
+      divId = "read-email-preview";
+    }
+
+    //SELECTED/UNSELECTED CHECK
+    //defaults, or unselected
+    let unitTestId = "unit-test-unselected-email-preview";
+    let pageStyle = "";
+    let buttonTextColor = styles.buttonUnselectedText;
+    let subjectIsSelected = styles.subjectUnselected;
+    if (this.props.isSelected) {
+      //if it is selected
+      unitTestId = "unit-test-selected-email-preview";
+      pageStyle = "page";
+      buttonBackgroundColor = styles.buttonSelectedBackground;
+      buttonTextColor = styles.buttonSelectedText;
+      subjectIsSelected = styles.subjectSelected;
+    }
+
+    //REPLIED TO/NOT REPLIED TO CHECK
+    //defaults, or no reply
+    let reply = <></>;
```

Comment:
This (and others) can easily be added to the return. In general, it's a best practice to exist in the return than outside of it.

For example, instead of `{reply}`, you could do
```
{ this.props.isRepliedTo && <i className="fas fa-sign-out-alt" /> }
```

Reply:
fixed in previous push